### DISCUSSION TITLE
Removate doesn't seem to always make a pr for outdated packages in package-json using pnpm package manager

### DISCUSSION THREAD

**Discussion by jpbnetley** [renovatebot/renovate#20078] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I want renovate to make pull requests as it detects outdated packages on my repo.
So when I run pmpm outdated, the packages listed there should get updated in a pull request from renovate.

In my test, if I run:
```cmd
joke-of-the-day git:(master) pnpm outdated
┌────────────┬─────────┬────────┐
│ Package    │ Current │ Latest │
├────────────┼─────────┼────────┤
│ jest (dev) │ 29.4.0  │ 29.4.1 │
├────────────┼─────────┼────────┤
│ vitest     │ 0.28.1  │ 0.28.3 │
```
There are outdated packages. But renovate doesn't detect them.

I know according to this discussion: https://github.com/renovatebot/renovate/discussions/19793 it is suggested that if the range is satisfied, the pr will not be made.

But if the project is not not being actively worked on. 
(so the package will not get updated and released if a developer does  update the lock file with `pnpm i`.)
 I ideally would like renovate to make the pull request for minor changes.

### What you tried so far.

I linked renovate by using the github app.
Then I moved the `renovate.json` file to my `.github` folder for consistency with where dependabot use to keep their config.

I updated the config to assign me in the pull request.
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "assignees": ["jpbnetley"]
}

```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
renovate currently don't update ranges when the range satisfies the new version. this will be changed in next major version.

then renovate will suggest lockfile updates for in range updates. checkout the v35 branch and merged commits 

- #19800

**Reply by jpbnetley** [reply]
aah ok perfect, thanks for the info :)
